test/Azure.Functions.Cli.Tests/E2E/Helpers/CliTester.cs

```diff
@@ -84,6 +84,11 @@ private static async Task InternalRun(string workingDir, RunConfiguration[] runC
                 {
                     var command = runConfiguration.Commands[i];
                     var exe = new Executable(_func, command, workingDirectory: workingDir);
+                    // Special case for dotnet commands
+                    if (command.StartsWith("dotnet", StringComparison.OrdinalIgnoreCase))
+                    {
+                        exe = new Executable("dotnet", command.Substring(7), workingDirectory: workingDir);
```

Comment:
Can you leave a comment in the file about why we are doing `Substring(7)`?

Reply:
added!